documentation/gitUsage.md

```diff
@@ -0,0 +1,26 @@
+## Steps for making a change
+ 
+ALL changes should be to
+visual-studio-code-extension-project
+ 
+GitHub Desktop
+ 
+1. have the repository locally
+> git clone https://github.com/visual-studio-code-extensions/visual-studio-code-extension-project.git
+2. make sure its updated
+> git checkout main
+> git pull
+3. make a new branch
+> git checkout -b <your GitHub alias>\<descriptive branch name>
+4. push the branch
+> git push
+if that git push you a command run that command
+
+6. 5. make your changes
+Edit code do stuff
+commit and push
+> git add  *
+> git commit -am "description"
+> git push
+6. Make a pull request
```

Comment:
`7` ?

Reply:
I copy pasted what you wrote and didn't double check it.😊